### ISSUE TITLE
Cannot open module file ‘eqstate.mod’ No such file or directory

### ISSUE THREAD

**Issue by tongyaopu** [fabm-model/fabm#34]
I successfully installed fabm with host=0d, so there is _~/local/fabm/0d/_ directory. But there was no /bin folder and no executable file under this directory. 

I proceed to compile it with fabm-0d following [this tutorial](https://github.com/fabm-model/fabm/wiki/0D-driver):

```
mkdir -p ~/build/fabm-0d && cd ~/build/fabm-0d
cmake ~/src/fabm/src/drivers/0d -DGOTM_BASE=~/src/gotm
make install
```

During the building process there was an error:
```
Scanning dependencies of target fabm0d
[ 98%] Building Fortran object CMakeFiles/fabm0d.dir/fabm0d.F90.o
~/src/fabm/src/drivers/0d/fabm0d.F90:17:8:

   17 |    use eqstate,only:rho_feistel
      |        1
Fatal Error: Cannot open module file ‘eqstate.mod’ for reading at (1): No such file or directory
compilation terminated.
```

Where should eqstate.mod be? During which step should it be generated?

Thank you!

**Comment by jornbr** [comment]
Thanks for flagging this. This is due to changes in the equation of state in the latest [developers'] version of GOTM, which fabm0d uses to calculate density. This will be fixed in a future FABM update. For now, the best workaround is to use the latest stable release of GOTM, which you can obtain with `git clone --recursive https://github.com/gotm-model/code.git -b v6.0 gotm6`. Then specify the resulting `gotm6` directory with `-DGOTM_BASE`.

**Comment by platipodium** [comment]
I see this has not been fixed as of FABM 2.0.1. @jornbr are you planning to work on this?

**Comment by jornbr** [comment]
This update to fabm0d will happen, but only when the GOTM changes have been incorporated into its next stable release (presumably v7.0). FABM will not track the development version of GOTM, as that is potentially still evolving. Tracking it would mean more work and bring the risk of specific commits of FABM only working with very specific commits of GOTM, which would be hard to document properly, and would not be very user friendly. We now more explicitly state that fabm0d is tied to GOTM's latest stable release: https://github.com/fabm-model/fabm/wiki/0D-driver.
